gamla/higher_order_test.py

```diff
@@ -1,7 +1,13 @@
 import asyncio
 import functools
+import zlib
```

Comment:
do we need to add this to the dependencies (setup.py)

Reply:
Yes it was not there - added it now

Reply:
sorry - on second look seems it is a built in library and does not need to be in setup